src/timepicker/timepicker.ts

```diff
@@ -133,6 +135,16 @@ export class NgbTimepicker implements ControlValueAccessor {
    */
   @Input() secondStep = 1;
 
+  /**
+   * To disable timepicker
+   */
+  @Input() disabled = false;
+
+  /**
+   * To make timepicker readonly
+   */
+  @Input() readonly = false;
```

Comment:
So we need to discuss semantics of `readonly`. The thing is that is you apply `readonly` to the entire control (inputs and buttons) than it is not different from `disabled` - as such there would be no point in having both. 

Looking at the implementation of http://angular-ui.github.io/bootstrap/#/timepicker I think that the intention is to mark inputs as readonly while still allowing people to click on buttons (chevrons, meridian). As such we should probably rename this property to `readonlyInputs` to better convey what we mean here.